src/tx-builder/services/DydxToken.ts

```diff
@@ -1,370 +1,370 @@
-            methodParameters: [this.treasuryAddresses.COMMUNITY_TREASURY_ADDRESS],
-          },
-          {
-            reference: 'communityTreasuryVesterBalance',
-            methodName: 'balanceOf',
-            methodParameters: [this.treasuryAddresses.COMMUNITY_TREASURY_VESTER_ADDRESS],
-          },
-        ],
-      },
-      {
-        reference: 'multicall',
-        contractAddress: this._multicallData.multicallAddress,
-        abi: multicallAbi,
-        calls: [
-          {
-            reference: 'timeLatest',
-            methodName: 'getCurrentBlockTimestamp',
-            methodParameters: [],
-          },
-        ],
-      },
-    ];
-
-    const multiRes: ContractCallResults = await this._multicallData.multicall.call(contractCallContext);
-
-    const tokenReturnContext = multiRes.results.token.callsReturnContext;
-    const multicallReturnContext = multiRes.results.multicall.callsReturnContext;
-
-    const transferRestriction: BigNumber = BigNumber.from(
-      tokenReturnContext.find(c => c.reference === 'transferRestriction')
-        ?.returnValues[0].hex,
-    );
-
-    const totalSupply: BigNumber = BigNumber.from(
-      tokenReturnContext.find(c => c.reference === 'totalSupply')
-        ?.returnValues[0].hex,
-    );
-
-    const rewardsTreasuryBalance: BigNumber = BigNumber.from(
-      tokenReturnContext.find(c => c.reference === 'rewardsTreasuryBalance')
-        ?.returnValues[0].hex,
-    );
-
-    const rewardsTreasuryVesterBalance: BigNumber = BigNumber.from(
-      tokenReturnContext.find(c => c.reference === 'rewardsTreasuryVesterBalance')
-        ?.returnValues[0].hex,
-    );
-
-    const communityTreasuryBalance: BigNumber = BigNumber.from(
-      tokenReturnContext.find(c => c.reference === 'communityTreasuryBalance')
-        ?.returnValues[0].hex,
-    );
-
-    const communityTreasuryVesterBalance: BigNumber = BigNumber.from(
-      tokenReturnContext.find(c => c.reference === 'communityTreasuryVesterBalance')
-        ?.returnValues[0].hex,
-    );
-
-    const currentTimestamp: BigNumber = BigNumber.from(
-      multicallReturnContext.find(c => c.reference === 'timeLatest')
-        ?.returnValues[0].hex,
-    );
-
-    if (currentTimestamp.lt(transferRestriction)) {
-      // Circulating supply is 0 before transfer restriction end
-      return '0.0';
-    }
-
-    // circulatingSupply = total supply - illiquid supply
-    // We consider all tokens owned by the treasuries + treasury vesters to be illiquid
-    // supply since it would require a governance vote + sufficient time for vesting to move the funds.
-    const circulatingSupplyWei: BigNumber = totalSupply
-      .sub(parseEther(LOCKED_ALLOCATION))
-      .sub(rewardsTreasuryBalance)
-      .sub(rewardsTreasuryVesterBalance)
-      .sub(communityTreasuryBalance)
-      .sub(communityTreasuryVesterBalance);
-
-    return formatEther(circulatingSupplyWei);
-  };
-}
+// import { formatEther, parseEther } from '@ethersproject/units';
```

Comment:
For my understanding why did this file have to be commented out?

Reply:
for some reason this crashes everything for me - haven't been able to figure out how to resolve

Reply:
uncommented now